packages/console/src/onboarding/App.tsx

```diff
@@ -23,9 +23,8 @@ import Congrats from './pages/Congrats';
 import SignInExperience from './pages/SignInExperience';
 import Welcome from './pages/Welcome';
 import { OnboardingPage, OnboardingRoute } from './types';
-import { getOnboardingPage, gtag } from './utils';
+import { gtag } from './utils';
```

Comment:
Is `getOnboardingPage` still needed? If not, we can remove it as well

Reply:
It's needed on other pages.